packages/orchestrator-ui-components/src/components/WfoTable/WfoAdvancedTable/getRowDetailData.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+    WfoAdvancedTableColumnConfig,
+    WfoAdvancedTableDataColumnConfigItem,
+} from '@/components/WfoTable/WfoAdvancedTable/types';
+import {
+    ColumnType,
+    WfoTableControlColumnConfigItem,
+} from '@/components/WfoTable/WfoTable';
+import { getTypedFieldFromObject } from '@/utils';
+
+export const getRowDetailData = <T extends object>(
+    selectedDataForDetailModal: T | undefined,
+    tableColumnConfig: WfoAdvancedTableColumnConfig<T>,
+) => {
+    if (!selectedDataForDetailModal) {
```

Comment:
What is the use case for this? Why just not call it if you don't want to use it in a detail modal?

Reply:
Good one, selectedDataForDetailModal is now only `T` and moved the undefined check to WfoAdvancedTable component